pkg/guestagent/guestagent_linux.go

```diff
@@ -333,3 +334,13 @@ func (a *agent) fixSystemTimeSkew() {
 		ticker.Stop()
 	}
 }
+
+func (a *agent) HandleInotify(event api.InotifyEvent) {
+	location := event.Location
+	if _, err := os.Stat(location); err == nil {
+		err := os.Chmod(location, event.Mode)
```

Comment:
Shouldn't we just use the mode statted in L340?

Reply:
I tried this, and at some point it actually corrupted my host system with file permissions as --------

Reply:
chmod looks dangerous anyway, let's just use `touch` and call it a day, even if it doesn't work with IntelliJ.

Reply:
For a long-term solution we have to submit a kernel patch to allow injecting inotify events from userspace.
Likely to take multiple years though.

Reply:
> chmod looks dangerous anyway

Exactly. That's why looking for possible alternates

> even if it doesn't work with IntelliJ

My fear is this will just be one-of usecase, this may even block the remote development possibilities from lima

Reply:
Changed to touch and also fixed the intellij issue. 
Previously events were getting dropped but now it look stable in macOS